### ISSUE TITLE
Allow name to be doom1.wad

### ISSUE THREAD

**Issue by Eeems** [LinusCDE/doomarkable#1]

**Comment by LinusCDE** [comment]
Yeah, for that I currently need to improve some of the library handling. That is more of an issue for [doomgeneric-rs](https://github.com/LinusCDE/doomgeneric-rs) probably.

**Comment by Eeems** [comment]
You should have a button on the sidebar to transfer this issue to that repository
